sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeriesId.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is composed of three primitive values that uniquely identifies a Time Series instance. The
+    /// keys that make up the Time Series Id are chosen when creating a Time Series Insights Gen2 environment through the Azure portal.
+    /// </summary>
+    /// <remarks>
+    /// A single Time Series Id can be composite of multiple properties are specified as Time Series Id at environment creation time.
+    /// The position and type of values must match Time Series Id properties specified on the environment and returned by Get Model Setting API.
+    /// </remarks>
+    public class TimeSeriesId
+    {
+        private (bool, object) _key1;
+        private (bool, object) _key2;
+        private (bool, object) _key3;
+
+        internal (bool, object) Key1 => _key1;
```

Comment:
Why have a field and a property for each key?

Reply:
oops. Properties are not needed.